include/app/pages/vehicle.hpp

```diff
@@ -4,11 +4,13 @@
 #include <QtWidgets>
 #include <QPluginLoader>
 
-#include "obd/obd.hpp"
+#include "canbus/socketcanbus.hpp"
+#include "obd/message.hpp"
+#include "obd/command.hpp"
 #include "app/widgets/selector.hpp"
 #include "app/widgets/dialog.hpp"
 
-typedef std::function<double(std::vector<double>, bool)> obd_decoder_t;
```

Comment:
im assuming this is why the 2 bottom "gauges" were removed right? The ones that required multiple obd pids to calculate the values?

Reply:
To be honest I had forgotten how I implemented this - but yes, that checks out. Because everything is callback based now, I hadn't found a good solution for gauges that needed multiple OBD pids. I'm sure that functionality can be added.

Reply:
i think keeping the simple implementation is fine for now, others can always get more creative by making their own plugins :)